src/ws-dropdown/ws-dropdown.js

```diff
@@ -185,11 +231,15 @@ export class WSDropdown extends Component {
    * @returns {Object}
    */
   createState(props) {
-    const state = {
-      text: this.getTextFromValue(props.value, props.text),
-      value: this.enrichItems(props.value),
-      items: this.enrichItems(props.items)
-    };
+    const items = this.enrichItems(props.items);
+    let value = props.value;
+    // Map value to the item to improve usability
```

Comment:
I was expecting to see `map` after reading the comment. Maybe it could be dropped completely, since the code is more precise about what's going on here.

Reply:
Haha It was my last change which introduced the test error. Was in hurry and now you see the result^^

Reply:
I would change it to a more descriptive one => "For better usability the value can be a primitive value matching a dropdown item value"